README.md

```diff
@@ -37,35 +37,7 @@ const allGravityData = () => {
                 allGfForm {
                     edges {
                         node {
-                            formId
-                            slug
-                            apiURL
-                            descriptionPlacement
-                            formFields {
-                                id
-                                label
-                                labelPlacement
-                                description
-                                descriptionPlacement
-                                type
-                                choices
-                                content
-                                errorMessage
-                                inputMaskValue
-                                isRequired
-                                visibility
-                                cssClass
-                                placeholder
-                                size
-                                defaultValue
-                                maxLength
-                            }
-                            button {
-                                text
-                            }
-                            confirmations {
-                                message
-                            }
+                            ...GravityFormComponent
```

Comment:
Where is this being imported from?

Reply:
Gatsby automatically imports graphql fragments, so you just need to use it via the `...GravityFormComponent` inside the gForb node. I'll fix the docs up to make them more clear.

Reply:
I thought that would be the case, but it isn't working for me. Throwing up an error.

Reply:
Hmm, I'm just building this and linking it as you indicated. Just checked again and seems to be working.

 When you build this, make sure the `fragment.js` file from source is getting moved to the `dist` folder for Gatsby to import.

Reply:
The file is there.

Its strange. I unlinked and relinked the package and still getting this error:

`There was an error in your GraphQL query:              
                                                       
The fragment "GravityFormComponent" does not exist.    
                                                       
   1 | import { useStaticQuery, graphql } from "gatsby"
   2 |                                                 
   3 | export const AllGravityData = () => {           
   4 |   const { allGfForm } = useStaticQuery(         
   5 |     graphql`                                    
   6 |       query {                                   
   7 |         allGfForm {                             
   8 |           edges {                               
   9 |             node {                              
> 10 |               ...GravityFormComponent           
     |               ^^^^^^^^^^^^^^^^^^^^^^^           
  11 |             }                                   
  12 |           }                                     
  13 |         }                                       
  14 |       }                                         
  15 |     `                                           
  16 |   )                                             
  17 |   return allGfForm                              
  18 | }                                               
  19 |                                                 `

What does your query look like in your project?

Reply:
Just like yours! I did just read somewhere that fragments are only available in components. So if you're using this in `gatsby-node.js` You will need to manually import it.

Reply:
Ah ha, that may be it! I'll check tomorrow morning! If so I'll add to the docs.

Reply:
So this is my current page set up:

'''
import React from "react"
import { graphql } from "gatsby"
import GravityFormForm from "gatsby-gravityforms-component"

export const query = graphql`
  query GravityFormQuery {
    allGfForm {
      edges {
        node {
          ...GravityFormComponent
        }
      }
    }
  }
`

const IndexPage = ({ data }) => {
  const formData = data.allGfForm
  return <GravityFormForm id={1} formData={formData} />
}

export default IndexPage
'''

As simple as it can be - still getting the fragment error. Could you take the above code and run your end? If it works then I can be sure it is something I have broken this end elsewhere!

I cannot get my head around why it isn't working, if its working for you!

Reply:
Okay, I set this up on my test project. I had the same issue, I finally realized the only difference is I never ran `yarn add gatsby-gravityform-component`, so I did, and all was fixed, so linking wasn't enough, I still had to run the install step, that made everything work.

Hopefully this fixes things for you as well.

Reply:
Yarn add downloads the most current package from NPM. It does not install the package on your local.

Reply:
Well it fixed the fragment missing, not sure why, maybe try releasing this branch under a tag on npm then install it that way. My guess is this has something to do with how Gatsby "finds" these fragments and you have to run the install to make it happen, or at least some form of the package has to be in node_modules. All I know is it fixed the problem for me.

Reply:
That is very strange, but it works! Nice one